packages/sdk/src/create.ts

```diff
@@ -38,16 +41,25 @@ const DEFAULT_NODE_REQUIREMENTS = {
 
 export { Libp2pComponents };
 
-const ensureShardingConfigured = (shardInfo: ShardingParams): void => {
-  if (
-    ("shards" in shardInfo && shardInfo.shards.length < 1) ||
-    ("contentTopics" in shardInfo && shardInfo.contentTopics.length < 1)
-  ) {
-    throw new Error(
-      "Missing required configuration options for static sharding or autosharding."
-    );
-  }
-};
+export async function createNodeFromApplicationInfo(
+  applicationInfo: Partial<ApplicationInfo>,
+  options?: ProtocolCreateOptions & WakuOptions & Partial<RelayCreateOptions>
+): Promise<LightNode> {
+  options = options ?? {};
+  const shardInfo = ensureShardingConfigured(applicationInfo);
+  options.shardInfo = shardInfo.shardingParams;
+  return _createNode(shardInfo.shardInfo, options);
+}
+
+export async function createNodeFromContentTopics(
```

Comment:
I think combining it with `createNode` is better. 
To me it seems reasonable to keep amount of `create*` function as low as possible. Ideally 2: light node, relay node (for later).

Reply:
updated code: https://github.com/waku-org/js-waku/pull/1809/files#diff-14b641615108f0f431feaf560af039c04e6018450f3b3a5178cbaa69094c6927R9